go-controller/pkg/node/openflow_manager.go

```diff
@@ -338,3 +327,25 @@ func bootstrapOVSFlows() error {
 
 	return nil
 }
+
+// localnetPortInfo returns the bridge name and the patch port name for the default network
+func localnetPortInfo(portsOutput string) (string, string) {
```

Comment:
I think @dceara fix is more reliable:
https://github.com/ovn-org/ovn-kubernetes/commit/bbb476b27cc32c74d2df686f3a7f0bd2d0889b5b

As long as we can be sure that only the default patch port will have the node name, it seems safer to filter on. testing with UDN it looks like it should work:

```
[root@ovn-worker ~]# ovs-vsctl list interface | grep to-br-int
options             : {peer=patch-breth0_default.l3.primary_ovn-worker-to-br-int}
options             : {peer=patch-localnet.network_ovn_localnet_port-to-br-int}
name                : patch-localnet.network_ovn_localnet_port-to-br-int
name                : patch-breth0_default.l3.primary_ovn-worker-to-br-int
options             : {peer=patch-breth0_ovn-worker-to-br-int}
name                : patch-breth0_ovn-worker-to-br-int
```

Reply:
But you can't assure a patch port will feature the bridge name. 

Localnet ports for secondary networks of `localnet` topology do not feature the node name. Check the test I've introduced. For instance, the [issue reported](https://issues.redhat.com/browse/OCPBUGS-37433) features this patch port: `patch-vlan2003_ovn_localnet_port-to-br-int` 

This would work with UDNs for layer3 networks. It will not with localnet.

Reply:
So, just to clarify the requirement:
- bootstrapOVSFlows() must be called **only for the  default network patch port**

The original regex was too loose, it would also match on user defined network patch ports.  https://github.com/ovn-org/ovn-kubernetes/commit/bbb476b27cc32c74d2df686f3a7f0bd2d0889b5b tries to fix that by updating the (already existing) regex.

However, I agree with @maiqueb, we should know through configuration what the bridge and node names are so we could replace the full ovs port dump + regex with the equivalent of:
```
ovs-vsctl find interface name=`patch-br-int-to-<bridge name>_<node>`
```
If the above returns a record (it can return max one record) then continue, get flows, ...

Reply:
We also wouldn't need this part anymore:
```
	// Get ofport of patchPort
	ofportPatch, stderr, err := util.GetOVSOfPort("get", "Interface", patchPort, "ofport")
	if err != nil {
		return fmt.Errorf("failed while waiting on patch port %q to be created by ovn-controller and "+
			"while getting ofport. stderr: %q, error: %v", patchPort, stderr, err)
	}
```
The first `find` call above would get us all the information about the patch port, including its `ofport`.